.github/workflows/ci-cd.yml

```
@@ -311,3 +311,21 @@ jobs:
         run: |
           gh release upload --repo vacanza/holidays ${{ github.ref_name }} dist/*
           gh release upload --repo vacanza/holidays ${{ github.ref_name }} holidays-*-sbom.json
+
+
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)